server/src/main/java/keywhiz/service/daos/SecretDAO.java

```diff
@@ -22,10 +22,8 @@
 import com.google.common.collect.Lists;
 import java.time.Instant;
 import java.time.OffsetDateTime;
+import java.util.*;
```

Comment:
I generally prefer importing individual classes rather than the full package.

Reply:
Agree, missed this. This must have been the IntelliJ auto-reorganizer